dev/KaairaGupta/winequality/data_loader.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+
+
+
+def load_split_preprocessed_data(i):
+
+    data_path = "../../../datasets/winequality.csv"
+    data = pd.read_csv(data_path)
+    count_false_recommendation = len(data["recommend"]==True) 
+    false_indices= np.array((data.recommend==True).index)
+    true_indices = np.array((data.recommend==False).index)
```

Comment:
It looks like the variable names `true_` and `false_` on lines 13-15 are the opposite of what the code does. Unless I'm missing something, I'd recommend renaming these to something more intuitive.

Reply:
Yes, it was a silly mistake from my side..I have taken care of it and am submitting the changes.